buildah.go

```diff
@@ -117,6 +118,9 @@ type BuilderOptions struct {
 	// ReportWriter is an io.Writer which will be used to log the reading
 	// of the source image from a registry, if we end up pulling the image.
 	ReportWriter io.Writer
+	// github.com/containers/image/types SystemContext to hold credentials
+	// for use with private repositories
+	CredsContext *types.SystemContext
```

Comment:
better using `SystemContext` even if we're just using a subset of the whole struct (`CredsContext` is misleading and one can think it's just for credentials)